src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix that filters flags
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
+    {
+        $constants = get_defined_constants();
+        foreach ($constants as $c => $v) {
+            if ('' !== $prefix && 0 !== strpos($c, $prefix) || ($value & $v) !== $v) {
+                unset($constants[$c]);
+            }
+        }
+
+        asort($constants);
+        $name = implode(' | ', array_flip($constants)) ?: 0;
```

Comment:
Ok, I understand better. But about these flags that have more bits that the defined constants, how show their dump? We could add these extra bits as numeric. Example:

```php
const A = 1;
const B = 2;
```

| int | dump
| --- | ---
| 1 | A
| 2 | B
| 3 | A │ B
| 4 | 4
| 5 | A │ 4
| 6 | B │ 4
| 7 | A │ B │ 4
| 8 | 8